data/core/units/monsters/Tusker_Tusklet.cfg

```diff
@@ -0,0 +1,99 @@
+#textdomain wesnoth-units
+[unit_type]
+    id=Tusklet
+    name= _ "Tusklet"
+    race=tusker
+    image="units/monsters/tusker/tusklet.png"
+    [standing_anim]
+        start_time=-50
+        [frame]
+            image="units/monsters/tusker/tusklet.png"
+            duration=50
+        [/frame]
+    [/standing_anim]
+    [standing_anim]
+        start_time=-50
+        terrain=Wo,Ww,Ww^Vm,Chs,Chw,Ss
+        [frame]
+            image="units/monsters/tusker/tusklet-water.png"
+            duration=50
+        [/frame]
+    [/standing_anim]
+#    ellipse="misc/ellipse-nozoc"
+    hitpoints=25
+    movement_type=orcishfoot
+    movement=4
+    experience=26
+    {AMLA_DEFAULT}
```

Comment:
It has advances_to and {AMLA_DEFAULT}

Reply:
I did a pretty minimal update to the original files, to get the images to work.  We'll have to make several adjustments to these .cfg files.